app/assets/stylesheets/utilities/layout.scss

```diff
@@ -9,3 +9,16 @@
 .justify-content-between {
   justify-content: space-between;
 }
+
+.flash-holder {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flash-message {
```

Comment:
If we modified this class to ".flash-message-notice", we could interpolate the flash :name in the view and easily extend this to have "flash-message-alert" or "flash-message-error" down the line.

Not vital for now but would remove the need to rework it later, your call.

Reply:
I'm happy to change this, I think it makes it clearer that it only handles notices. 

I'm going to look again at how this interacts with the other partial as well, see if I can make something nicer (maybe as a separate PR)